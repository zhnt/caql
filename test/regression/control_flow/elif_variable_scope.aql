// Test elif with variable scope
let global_var = "global"
if 1 == 2 {
    let if_var = "if_block"
    print("In if: " + if_var)
} elif 2 == 2 {
    let elif_var = "elif_block"
    print("In elif: " + global_var)
    print("In elif: " + elif_var)
} else {
    print("In else")
}
print("After elif: " + global_var)
