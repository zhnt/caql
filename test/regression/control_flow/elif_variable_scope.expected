In elif: global
In elif: elif_block
After elif: global
